ExpressJS/utils/sequilize.js

```diff
@@ -1,24 +1,15 @@
-require('dotenv').config({ path: './utils/dev.env' });
 const fs = require('fs');
 const { Sequelize, Op, Model, DataTypes } = require('sequelize');
 
-
 const dbConnectionType = process.env.DB_CONNECTION_TYPE;
-console.log('POSTGRES_USER:', process.env.POSTGRES_USER);
-console.log('POSTGRES_PASSWORD:', process.env.POSTGRES_PASSWORD);
-console.log('POSTGRES_HOST:', process.env.POSTGRES_HOST);
-console.log('POSTGRES_PORT:', process.env.POSTGRES_PORT);
-console.log('DB_CONNECTION_TYPE:', process.env.DB_CONNECTION_TYPE);
 
 let sequelize;
 if (dbConnectionType === 'dev_db') {
-  sequelize = new Sequelize(
-    `postgres://${process.env.POSTGRES_USER}:${process.env.POSTGRES_PASSWORD}@${process.env.POSTGRES_HOST}:${process.env.POSTGRES_PORT}/waect`,
-    { // Sequlieze connection for postgres
-      logging: console.log
+  sequelize = new Sequelize('waect', 'user', 'pass', {
```

Comment:
Check my suggestion in the other PR.
you can just pass the URL here and change it in the .env before run

Reply:
> Then we can use the same .env! It is nice if we could keep only one env var for the db conneciton

Reply:
done